llvm/test/Transforms/CodeGenPrepare/RISCV/noop-copy-sink.ll

```diff
@@ -0,0 +1,30 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
+; RUN: opt -S -passes='require<profile-summary>,function(codegenprepare)' -mtriple=riscv64 %s \
```

Comment:
profile-summary doesn't look relevant here...

Reply:
You'd hope not. I got this from the and-mask-sink.ll test in the same directory. Removing the profile-summary does indeed cause a crash, from the isFunctionHotINCallGraph call in CodeGenPrepare::_run which is odd.

It seems all tests in tests/Transform/CodeGenPrepare have the same profile-summary requirement.

Reply:
Seems the ugly workaround was noted in #75380 when CodeGenPrepare was ported to the new pass manager.